openfisca_france/scripts/parameters/baremes_ipp/README.md

```diff
@@ -10,51 +10,88 @@ L'IPP produit des fichiers au format XLSX appelés « Barèmes IPP » :
 
 Ces barèmes IPP sont très complets et il est intéressant pour OpenFisca de profiter de leur contenu.
 
-## Conversion XLSX vers XML
+Les scripts décrits ici permettent d'importer les barèmes IPP (fournis sous forme de tableurs au format XLSX) dans [les fichiers de paramètres](https://doc.openfisca.fr/coding-the-legislation/legislation_parameters.html) XML d'OpenFisca France.
 
-OpenFisca définit ses paramètres dans un format XML décrit [ici](https://doc.openfisca.fr/coding-the-legislation/legislation_parameters.html).
+## Imports des barèmes IPP dans OpenFisca
 
-L'IPP fournit ses paramètres sous forme de tableurs au format XLSX.
+### En local
 
-Le script [`convert_ipp_xlsx_to_openfisca_xml.py`](./convert_ipp_xlsx_to_openfisca_xml.py) prend en charge la conversion de XLSX vers XML.
+>**Attention, ce script ne semble fonctionner que sous Linux, et pas sous macOS. Voir la [discussion](https://github.com/openfisca/openfisca-france/pull/746#issuecomment-305123915) sur le sujet.**  
+> **Si vous avez accès au serveur OpenFisca-France, vous pouvez [executer le script sur ce-dernier](#sur-le-serveur-openfisca)**
 
->**Attention, ce script ne semble fonctionner que sous Linux, et pas sous macOS. Voir la [discussion](https://github.com/openfisca/openfisca-france/pull/746#issuecomment-305123915) sur le sujet.**
+#### Installation
 
-Ce script nécessite le paquet [`xlrd`](https://pypi.python.org/pypi/xlrd). Pour l'installer :
+Ce script nécessite des dépendances supplémentaires. Pour les installer :
 
 ```sh
-pip install xlrd
-# Ou bien utilisez l'extra-require comme ceci : pip install --editable ".[baremes_ipp]"
+pip install --editable .[baremes_ipp]
 ```
 
 Il faut également installer `gnumeric` :
 - sous Debian : `sudo apt install gnumeric`
-- sous macOS : `brew install gnumeric`
+
+#### Utilisation
 
 Se placer dans le répertoire racine d'OpenFisca-France, là où se trouve le fichier `setup.py` :
 
 ```sh
-./openfisca_france/scripts/parameters/baremes_ipp/convert_ipp_xlsx_to_openfisca_xml.py
+./openfisca_france/scripts/parameters/baremes_ipp/convert_ipp_xlsx_to_openfisca_xml.py --ref-ipp 2c6936d6 --merge
 ```
 
-Le script génère un nouveau répertoire dans `/tmp` à chaque exécution, dont une partie du nom est aléatoire. Ce répertoire n'est pas effacé par le script. Par exemple :
+> L'argument `--ref-ipp` permet de spécifier une version spécifique des [fichiers `XLSX` publiés par l'IPP](https://framagit.org/french-tax-and-benefit-tables/ipp-tax-and-benefit-tables-xlsx/repository/archive.zip) à utiliser.
```

Comment:
Préciser qu'il s'agit de version git ? En particulier parce qu'on désigne des fichiers Excel.
Et, dans cette phrase générique, mettre plutôt un lien vers le [dépôt IPP](https://git.framasoft.org/french-tax-and-benefit-tables/ipp-tax-and-benefit-tables-xlsx) ?